docker-compose.yml

```diff
@@ -1,21 +1,32 @@
-db:
-  image: postgres
-elasticsearch:
-  image: elasticsearch:2
-rabbitmq:
-  image: rabbitmq
-web:
-  image: ccnmtl/locustempus
-  environment:
-    - APP=plexus
-    - SECRET_KEY=dummy-secret-key
-    - SETTINGS=settings_compose
-  command: manage runserver 0.0.0.0:8000
-  volumes:
-    - .:/app/
-  ports:
-    - "8000:8000"
-  links:
-    - db
-    - elasticsearch
-    - rabbitmq
+services:
+  db:
+    image: postgis/postgis:15-3.3
+    volumes:
+      - ./data/db:/var/lib/postgresql@13/data
```

Comment:
The `@13` can be removed here, now that the image is using version 15, and I don't think you need to specify the version number in this path.

Reply:
It requires a pin, otherwise there is a version mismatch.
<img width="1249" alt="Screenshot 2023-08-17 at 1 28 05 PM" src="https://github.com/ccnmtl/locustempus/assets/107073665/204207fa-0811-4f0f-9f9e-a3d9976cb9e5">

Reply:
It looks like the database on your filesystem needs to be refreshed with the new postgis - try removing the `data` directory and running the `docker compose up` again.